src/more-infos/more-info-climate.html

```diff
@@ -322,19 +323,17 @@
     return 0.5;
   }
 
-  computeTargetTempHidden(stateObj) {
-    return !stateObj.attributes.temperature &&
-        !stateObj.attributes.target_temp_low &&
-        !stateObj.attributes.target_temp_high;
+  supportsTemperature(stateObj) {
+    return (stateObj.attributes.supported_features & 1) !== 0;
   }
 
-  computeIncludeTempRangeControl(stateObj) {
-    return stateObj.attributes.target_temp_low ||
-        stateObj.attributes.target_temp_high;
+  supportsTemperatureRange(stateObj) {
+    return (stateObj.attributes.supported_features & 4) !== 0;
```

Comment:
To preserve previous behavior of (high or low) compare with "6"

Reply:
Will fix that next, but I'm still unsure if using supported_features is correct because it breaks all climate demo components display.